lib/LedgerSMB/Scripts/budgets.pm

```diff
@@ -179,7 +179,7 @@ sub view_budget {
         my $row = {};
         $row->{description} = $line->{description};
         if ($line->{amount} < 0 ) {
-            $row->{debit} = $line->{amount} * -1;
+            $row->{debit} = -$line->{amount};
```

Comment:
Here, the intent of the code is to make the value positive. I'd say that the `abs()` (https://perldoc.perl.org/functions/abs.html) function should do exactly that.